classLab.js

```diff
@@ -0,0 +1,159 @@
+//1
+
+class Person {
+    constructor(firstName, middleName, lastName){
+        this.firstName = firstName
+        this.middleName = middleName
+        this.lastName = lastName
+    }
+    fullName(){
+        return this.firstName + " " + this.middleName + " " + this.lastName
```

Comment:
Its correct but try to look into string interpolation to accomplish this.You might prefer that way